app/assets/stylesheets/govuk_publishing_components/components/_big-number.scss

```diff
@@ -5,12 +5,8 @@
 
 .gem-c-big-number__value {
   @include govuk-font($size: false, $weight: bold);
-  font-size: 80px;
```

Comment:
I think this ship sailed a long time ago, but removing the `font-size` in `px` throughout means you're outputting font sizes _only_ in rem, so [browsers that don't support rem](https://caniuse.com/rem) will not have font sizes set any more.

Reply:
Oh I see, so I should leave the px font-size, and only take out the if statement.

Reply:
Fixed.